docs/standard/library.md

```diff
@@ -124,6 +139,9 @@ You can see the set of PCL profiles that are compatible with the .NET Standard:
 | Profile157 .NET Portable Subset (Windows 8.1, Windows Phone 8.1, Windows Phone Silverlight 8.1) | 1.0 |
 | Profile259 .NET Portable Subset (.NET Framework 4.5, Windows 8, Windows Phone 8.1, Windows Phone Silverlight 8) | 1.0 |
```

Comment:
I suggest to leave the note out. The correct TFM representation of the profile isn't that important and will confuse it even more, IMHO.